docs/contributing/contributing-rules.md

```diff
@@ -38,17 +37,26 @@ This workflow automatically creates a pull request in the GitHub [Semgrep Regist
 
 You can also publish rules as private rules outside of Semgrep Registry. These rules are not included in the Semgrep Registry, but they are accessible to your Semgrep organisation. See the [Private rules](/writing-rules/private-rules) documentation for more information.
 
-## Contributing through GitHub
+## Contribute through GitHub
 
-Fork our repository and make a pull request. Sign our Contributor License Agreement (CLA) on GitHub before Semgrep, Inc. can accept your contributions. Make a pull request to the [Semgrep Registry](https://github.com/semgrep/semgrep-rules) with two files:
-1. The semgrep pattern (as YAML file).
-2. The test file (with the file extension of the language or framework). The test file must contain at least one true positive and one true negative test case to be approved. See the [Tests](#tests) section of this document for more information.
+Fork the [<i class="fas fa-external-link fa-xs"></i> semgrep](https://github.com/semgrep/semgrep) repository and make a pull request. Sign the Contributor License Agreement (CLA) on GitHub; this is required before Semgrep can accept your contributions. Make a pull request to the [Semgrep Registry](https://github.com/semgrep/semgrep-rules) with two files:
```

Comment:
I'm confused by this section; shouldn't the user fork the semgrep-registry repo and open a PR there? Or, am I missing something?

Reply:
Good catch. There are two separate instructions here that seem to have been squished together. I git blamed and it looks like it was work from years ago.

1. You can fork `semgrep-rules`, merge your rule, and make a PR to bring your changes upstream.
2. You can make a PR directly in `semgrep-rules` - this is the more straightforward way, and I saw a contributor just do it recently, so I'll use that.